### ISSUE TITLE
Error during upgrade from 1.8.0 to 1.8.1 due to scope entitlements

### ISSUE THREAD

**Issue by zinus92** [indigo-iam/iam#600]
```
SQL State  : 23000
Error Code : 1062
Message    : Duplicate entry 'entitlements' for key 'scope'
Location   : db/migration/mysql/V90__fix_eduperson_entitlement_scope.sql (/indigo-iam/file:/indigo-iam/WEB-INF/lib/iam-persistence-1.8.1.jar!/db/migration/mysql/V90__fix_eduperson_entitlement_scope.sql)
Line       : 1
Statement  : INSERT INTO system_scope(scope, description, icon, restricted, default_scope, structured, structured_param_description)
  VALUES
  ('entitlements', 'Access to entitlements information', null, false, false, false, null)
```

**Comment by rmiccoli** [comment]
Hi,

did you go through v1.8.1-rc first? 
We inserted the migration `V90__fix_eduperson_entitlement_scope.sql` in v1.8.1-rc1. So if you roll back from v1.8.1-rc to v1.8.0 the  `entitlements` database entry  remains and causes duplication problems when you then upgrade to v1.8.1.

**Comment by zinus92** [comment]
Hi,

> did you go through v1.8.1-rc first?

No, I did a direct jump from 1.8.0 to 1.8.1.

> The workaround is to delete that scope from the DB table and then re-try to upgrade.

We applied this solution and it works.

**Comment by enricovianello** [comment]
Closed as a known issue about upgrade from 1.7.2/1.8.0 to 1.8.1, downgrade and then re-upgrade to 1.8.1
